CHANGELOG.md

```diff
@@ -1,10 +1,64 @@
+
+- api: Add 'CreatedAt', 'StartedAt' in pod's info returned by api service. ([#2377](https://github.com/coreos/rkt/pull/2377)).
+
+#### Improved documentation
+
+- Minor documentation fixes ([#2413](https://github.com/coreos/rkt/pull/2413), [#2395](https://github.com/coreos/rkt/pull/2395), [#2231](https://github.com/coreos/rkt/pull/2231)).
+
+- functional tests: Add new test with systemd-proxyd ([#2257](https://github.com/coreos/rkt/pull/2257)). Adds a new test and documentation how to use systemd-proxyd with rkt pods.
+
+#### Bug fixes
+
+- kvm: refactor volumes support ([#2328](https://github.com/coreos/rkt/pull/2328)). This allows users to share regular files as volumes in addition to directories.
+
+- kvm: fix rkt status ([#2415](https://github.com/coreos/rkt/pull/2415)). Fixes a regression bug were `rkt status` was no longer reporting the pid of the pod when using the kvm flavor.
+
+- Build actool for the *build* architecture ([#2372](https://github.com/coreos/rkt/pull/2372)). Fixes a cross compilation issue with acbuild.
+
+- rkt: calculate real dataDir path ([#2399](https://github.com/coreos/rkt/pull/2399)). Fixes garbage collection when the data directory specified by `--dir` contains a symlink component.
+
+- stage1/init: fix docker volume semantics ([#2409](https://github.com/coreos/rkt/pull/2409)). Fixes a bug in docker volume semantics when rkt runs with the option `--pod-manifest`. When a Docker image exposes a mount point that is not mounted by a host volume, Docker volume semantics expect the files in the directory to be available to the application. This was partially fixed in rkt 1.3.0 via ([#2315](https://github.com/coreos/rkt/pull/2315)) but the bug remained when rkt runs with the option `--pod-manifest`. This is now fully fixed.
+
+- rkt/image: check that discovery labels match manifest labels ([#2311](https://github.com/coreos/rkt/pull/2311)).
+
+- store: fix comment + add link in db test ([#2407](https://github.com/coreos/rkt/pull/2407)).
+
+- store: fix multi process with multi goroutines race on db. ([#2391](https://github.com/coreos/rkt/pull/2391)).
+
+- kvm: fix pid vs ppid usage ([#2396](https://github.com/coreos/rkt/pull/2396)).
+
+- kvm: Fix macvtap naming issue, closes #2023 ([#2181](https://github.com/coreos/rkt/pull/2181)).
+
+- tests: TestRktListCreatedStarted: fix timing issue ([#2366](https://github.com/coreos/rkt/pull/2366)).
+
+- rkt/image: remove redundant quotes ([#2379](https://github.com/coreos/rkt/pull/2379)).
+
+- prepare: Support 'ondisk' verification skip ([#2376](https://github.com/coreos/rkt/pull/2376)). Prior to this commit, rkt prepare would check the ondisk image even if the --insecure-options=ondisk flag was provided. This corrects that.
+
+#### Other changes
+
+- tests: TestDockerVolumeSemantics: more tests with symlinks ([#2394](https://github.com/coreos/rkt/pull/2394)).
+
+- rkt: Improve build shell script ([#2320](https://github.com/coreos/rkt/pull/2320)).
```

Comment:
- rkt: Improve build shell script used in [continuous integration](https://github.com/coreos/rkt/blob/master/tests/README.md) ([#2394](https://github.com/coreos/rkt/pull/2394)).